actionpack/test/journey/nodes/ast_test.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "abstract_unit"
+
+module ActionDispatch
+  module Journey
+    module Nodes
+      class TestAst < ActiveSupport::TestCase
```

Comment:
I think we can take this opportunity to improve the unit test coverage of the AST class